### PR TITLE
WIP update integration example

### PR DESCRIPTION
**Pull request by soanvig**
This is WIP, so it's not organized. It's just draft code to run the application at all

### CODE REVIEW

File: packages/origin-api/src/app.module.ts [hunk 1]
```diff
@@ -1,14 +1,35 @@
 import {
   AppModule as ExchangeModule,
   entities as ExchangeEntities,
+  IExchangeConfigurationService,
+  IOrderMapperService,
 } from "@energyweb/exchange";
```
Comment (josipbagaric) [inline packages/origin-api/src/app.module.ts]: Does it help if you use this instead?
```js
import {
  AppModule as ExchangeModule,
  entities as ExchangeEntities,
} from "@energyweb/exchange-irec";
```

Then you might not need to specify the providers at the end of the file.

Comment (soanvig) [reply packages/origin-api/src/app.module.ts]: I believe entities should be imported from general `exchange` right? ;)

Anyway, no, unfortunately now I'm missing `IExternalDeviceService` implementation, which is required by irec `ProductService` in irec.

Neither irec or base exchange implement that interface. In demo app it's implemented manually.

Comment (soanvig) [reply packages/origin-api/src/app.module.ts]: And this manual implementation is based on `@energyweb/origin-device-registry-irec-form-api` DeviceService.

**Comment by soanvig** [comment]
I'm merging that into the same branch but in the original repository

**Pull request merged**